packages/core-utils/src/__tests__/__mocks__/config.json

```diff
@@ -0,0 +1,55 @@
+{
+  "homeTimezone": "America/New_York",
+  "routingTypes": [
+    {
+      "key": "ITINERARY",
+      "text": "Exact Time"
+    }
+  ],
+  "modes": {
+    "transitModes": [
+      {
+        "mode": "BUS",
+        "label": "Bus"
+      },
+      {
+        "mode": "RAIL",
+        "label": "Rail"
+      },
+      {
+        "mode": "SUBWAY",
+        "label": "Subway"
+      },
+      {
+        "mode": "TRAM",
+        "label": "Trolleys"
+      }
+    ],
+    "accessModes": [
+      {
+        "mode": "BICYCLE",
+        "label": "Transit + Personal bike"
+      },
+      {
+        "mode": "BICYCLE_RENT",
+        "label": "Transit + Bikeshare"
+      }
+    ],
+    "exclusiveModes": ["WALK", "BICYCLE"],
+    "bicycleModes": [
+      {
+        "mode": "BICYCLE",
+        "label": "Own Bike"
+      },
+      {
+        "mode": "BICYCLE_RENT",
+        "label": "Bikeshare"
+      }
+    ]
+  },
+  "dateTime": {
+    "timeFormat": "h:mm a",
+    "dateFormat": "MM/DD/YYYY",
+    "longDateFormat": "MMMM D, YYYY"
+  }
+}
```

Comment:
I think there should be some `defaultQueryParams` that should be mentioned in the test.

Reply:
The tests clone/modify this default config to have the `defaultQueryParams`. I'd rather have a standard, multi-purpose config in place for multiple tests and modify that as needed for each test.